commands/remove.ts

```diff
@@ -19,25 +18,23 @@ exports.command = 'remove <path>';
 exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.handler = async options => {
-  const { path: hsPath } = options;
+  const { path: hsPath, account } = options;
 
   await loadAndValidateOptions(options);
 
-  const accountId = getAccountId(options);
-
-  trackCommandUsage('remove', null, accountId);
+  trackCommandUsage('remove', null, account);
 
   try {
-    await deleteFile(accountId, hsPath);
-    logger.log(i18n(`${i18nKey}.deleted`, { accountId, path: hsPath }));
+    await deleteFile(account, hsPath);
+    logger.log(i18n(`${i18nKey}.deleted`, { account, path: hsPath }));
```

Comment:
Should be `accountId: account` here and in the `ApiErrorContext` below

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1212/commits/11c8c119479c484027f71dffa1ad97d64951d2d8